modules/nodes-pods-configuring-reducing.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+// * nodes/nodes-cluster-pods-configuring
+
+[id="nodes-pods-configuring-reducing_{context}"]
+= Reducing pod timeouts when using persistent volumes with high file counts
+
+
+If a storage volume contains many files (~1,000,000 or greater), you might experience pod timeouts.
+
+This can occur because, when volumes are mounted, {product-title} recursively changes the ownership and permissions of the contents of each volume in order to match the `fsGroup` specified in a pod's `securityContext`. For large volumes, checking and changing the ownership and permissions can be time consuming, resulting in a very slow pod startup. 
+
+You can reduce this delay by applying one of the following workarounds:
+
+* Use an SCC to skip the SELinux relabeling for a volume.
+
+* Use the Cluster Resource Override Operator (CRO) to automatically apply an SCC to skip the SELinux relabeling.
+
+* Use the `fsGroupChangePolicy` field inside a security context constraint (SCC) to control the way that {product-title} checks and manages ownership and permissions for a volume.
```

Comment:
this is the third time SCC appears, should swap the spelled out version up earlier :)